docs/migration-guides/sfa-ios-v8-to-v9.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: SFA iOS SDK - v8 to v9
+description: "SFA iOS SDK - v8 to v9 | Documentation - Web3Auth"
+sidebar_label: v8 to v9
+---
+
+This migration guide provides steps for upgrading from version v8 to v9 of the SFA iOS SDK. The
+guide outlines significant changes and enhancements.
+
+## Breaking Changes
+
+### Web3AuthNetwork Changes
+
+In v9, we have removed the nest enum and refactored the Web3AuthNetwork. Please checkout the table
+below for the changes.
+
+| Old Value                    | New Value         |
+| ---------------------------- | ----------------- |
+| .sapphire(.SAPPHIRE_MAINNET) | .SAPPHIRE_MAINNET |
+| .sapphire(.SAPPHIRE_DEVNET)  | .SAPPHIRE_DEVNET  |
+| .legacy(.MAINNET)            | .MAINNET          |
+| .legacy(.TESTNET)            | .TESTNET          |
+| .legacy(.CYAN)               | .CYAN             |
+| .legacy(.AQUA)               | .AQUA             |
+
+### SFAParams Changes
+
+In v9, we try to improve the developer experience by renaming the `SFAParams` to `Web3AuthOptions`.
+It has been renamed to align with naming conventions across Web3Auth SDKs. Along with this, we have
+renamed couple of constructor parameters.
+
+- `web3AuthClientId` is reanmed to `clientId`.
+- `network` is reanmed to `web3AuthNetwork`.
+
+```swift
+// remove-start
+let singleFactorAuthArgs = SFAParams(
+    web3AuthClientId: "YOUR_WEB3AUTH_CLIENT_ID",
+    network: .sapphire(.SAPPHIRE_MAINNET)
+)
+// remove-end
+
+// add-start
+let web3AuthOptions = Web3AuthOptions(
+    clientId: "YOUR_WEB3AUTH_CLIENT_ID",
+    web3AuthNetwork: .SAPPHIRE_MAINNET
+)
+// add-end
+
+let singleFactoreAuth = SingleFactorAuth(
+    // remove-next-line
+    params: SFAParams
+    // add-next-line
+    params: web3AuthOptions
+)
+```
+
```

Comment:
There are more changes to the params that need to be listed here.

Reply:
Added link to the main docs